Multisig/Features/Security/KeychainItem.swift

```diff
@@ -68,9 +68,6 @@ enum KeychainItem {
 
             if let context = LAContext(password: password) {
                 result[kSecUseAuthenticationContext] = context
-            }  else {
```

Comment:
what if keychain item has been initialised when device password was not set? Wouldn't this edge-case lead to problems?